stdlib/2/pydoc.pyi

```diff
@@ -15,11 +16,11 @@ __version__ = ...  # type: str
 __credits__ = ...  # type: str
 
 def pathdirs() -> List[str]: ...
-def getdoc(object: object) -> str: ...
-def splitdoc(doc) -> Tuple[str, str]: ...
+def getdoc(object: object) -> AnyStr: ...
```

Comment:
AnyStr as just a return value doesn't make sense because there's nothing to constrain the type variable.